### ISSUE TITLE
Best settings for slurum

### ISSUE THREAD

**Issue by nbutyrate** [cruizperez/MicrobeAnnotator#41]
Hi,

 I am trying to annotate 12 microbial genomes using MA installed at HPC (slurum), (using blast). following are the current options I am using, but it's been running for days, I was wondering if you could help me in getting this speed up or highlight my mistake/s

microbeannotator -i  1.fasta  2.fasta  3.fasta  4.fasta  5.fasta  6.fasta  7.fasta  8.fasta  9.fasta  10.fasta  11.fasta  12.fasta -d //MicrobeAnnotator/2.0.5/db -o MicrobeAnnotator_ALL -m blast -p 5 -t 15

**Comment by cruizperez** [comment]
I think the best option you have to speed up things is to use Diamond. It basically retrieves the same results but way faster. Blast will take a really long time, especially if you use Trembl and Refseq. Let me know if I can help with anything else.

**Comment by nbutyrate** [comment]
Thanks for your reply,

my only goal is to get "metabolic_summary__module_completeness.tab", I don't need Trembl and Refseq, is there any way (removing or adding an option in any of the parameters) to not to query the genomes against refseq and Trembel? and only ko?

**Comment by cruizperez** [comment]
Yes, if you use the --light flag it will only use kofam and swissprot
